lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,26 @@
+#each_char
+#palindrome_permutation?("hello") => false
+#palindrome_permutation?("carrace") => true #because racecar is a palindrome
+#Write a method which takes a string as an argument and returns true if the letters could be re-arranged into a palindrome.
+
+
+def palindrome_permutation?(my_string)
```

Comment:
This also doesn't work, it's checking to see if the string **is** a palindrome, **not** to see if the string is a permutation of a palindrome.  

The hint is that you should use a hash.